apps/app/src/features/notes/components/note-item.tsx

```
@@ -0,0 +1,44 @@
+        <Card className="hover:bg-muted/50 h-full transition-colors cursor-pointer border-none shadow-none bg-muted/20">
+          <CardHeader className="flex flex-row items-start justify-between space-y-0">
+            <CardTitle className="text-lg line-clamp-1 pr-8">
+              {note.title}
+            </CardTitle>
+          </CardHeader>
+          <CardContent className="space-y-4">
+            {note.content && (
+              <div
+                className="text-sm text-muted-foreground line-clamp-3 prose prose-sm prose-muted max-w-none 
+                [&_p]:mb-0 [&_ul]:my-0 [&_ol]:my-0 [&_li]:my-0"
+                dangerouslySetInnerHTML={{ __html: note.content }}
```

Biome (2.4.7)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)